### PR TITLE
Updating the edge tutorial following the latest changes to the edge installation scripts

### PR DESCRIPTION
**Pull request by SStorm**
## Summary of the changes / Why this is an improvement

Some pre-requisites are no longer required (handled by the script) and clarified bare-metal usage.

## Checklist

 - [x] [CLA](https://crate.io/community/contribute/cla/) is signed

### CODE REVIEW

File: docs/edge.rst [hunk 1]
```diff
@@ -48,29 +48,32 @@ Kubernetes cluster, one that meets the following requirements:
 
 .. rst-class:: open
 
-* It must contain at least three nodes (for high availability);
+* It must contain at least three nodes (for high availability).
+  You can also run development workloads on a single-node cluster, however please note
```
Comment (matthijskrul) [inline docs/edge.rst]: Line too long. Please wrap all lines after 80 characters

File: docs/edge.rst [hunk 2]
```diff
@@ -48,29 +48,32 @@ Kubernetes cluster, one that meets the following requirements:
...
+  that you will only be able to provision single-node CrateDB "clusters";
```
Comment (matthijskrul) [inline docs/edge.rst]: I'm not sure I understand this - what is the 'however' here?

File: docs/edge.rst [hunk 3]
```diff
@@ -48,29 +48,32 @@ Kubernetes cluster, one that meets the following requirements:
...
+* A storage class for persistent data.
 
 Beyond this, using the CrateDB Cloud stack requires creating a CrateDB Cloud
 account and an organization, which will become the owner of the Edge region in
 which the cluster can be deployed. One must also access the CrateDB Cloud
 Console in order to deploy the cluster itself, using the provided script. These
 steps will be explained below.
 
+.. NOTE::
+    A special note about Bare Metal Kubernetes clusters. CrateDB Edge should work on
```
Comment (matthijskrul) [inline docs/edge.rst]:
```suggestion
    A special note about bare metal Kubernetes clusters. CrateDB Edge should work on
```

File: docs/edge.rst [hunk 4]
```diff
@@ -48,29 +48,32 @@ Kubernetes cluster, one that meets the following requirements:
...
+    any bare-metal cluster, however the CrateDBs running within require a Load Balancer
```
Comment (matthijskrul) [inline docs/edge.rst]:
```suggestion
    any bare metal cluster, but the CrateDB instances running within require a load balancer
```

File: docs/edge.rst [hunk 5]
```diff
@@ -48,29 +48,32 @@ Kubernetes cluster, one that meets the following requirements:
...
+    for outside access. If you do not have a Load Balancer (such as `MetalLB`_), you can
```
Comment (matthijskrul) [inline docs/edge.rst]:
```suggestion
    for outside access. If you do not have a load balancer (for example `MetalLB`_), you can
```

File: docs/edge.rst [hunk 6]
```diff
@@ -48,29 +48,32 @@ Kubernetes cluster, one that meets the following requirements:
...
+    still access the CrateDB Clusters within, however will need to figure out the node
```
Comment (matthijskrul) [inline docs/edge.rst]:
```suggestion
    still access the CrateDB clusters within, but you will need to figure out the node
```

File: docs/edge.rst [hunk 7]
```diff
@@ -306,6 +283,11 @@ Once the cluster is fully deployed, it can be accessed through the `CrateDB
 Admin UI`_ using the username and password you have defined and the URL of your
 cluster.
 
+.. NOTE::
+    If your Kubernetes cluster does not provide a Load Balancer with an external IP
```
Comment (matthijskrul) [inline docs/edge.rst]:
```suggestion
    If your Kubernetes cluster does not provide a load balancer with an external IP
```

File: docs/edge.rst [hunk 8]
```diff
@@ -306,6 +283,11 @@ Once the cluster is fully deployed, it can be accessed through the `CrateDB
...
+    address, you will not be able to access your cluster from the Cloud Console.
```
Comment (matthijskrul) [inline docs/edge.rst]:
```suggestion
    address, you will not be able to access your cluster from the CrateDB Cloud Console.
```

File: docs/edge.rst [hunk 9]
```diff
@@ -349,4 +331,5 @@ Cloud Edge from your local Kubernetes cluster.
 .. _ingress-nginx: https://github.com/kubernetes/ingress-nginx
 .. _installation instructions: https://kubernetes.github.io/ingress-nginx/deploy/
 .. _subscription plan: https://crate.io/docs/cloud/reference/en/latest/subscription-plans.html
-.. _support email: support@crate.io
\ No newline at end of file
+.. _support email: support@crate.io
+.. _MetalLB: https://metallb.universe.tf/
```
Comment (matthijskrul) [inline docs/edge.rst]: Links are in alphabetic order

File: docs/edge.rst (hunk 1 above)
Comment (matthijskrul) [inline docs/edge.rst]:
```suggestion
  You can also run development workloads on a single-node cluster. Note, however,
```

**Review by matthijskrul — CHANGES_REQUESTED** [review]

File: docs/edge.rst (hunk 2 above)
Comment (SStorm) [reply docs/edge.rst]: I think you changed that in one of the suggestions and I applied it.

File: docs/edge.rst [hunk 10]
```diff
@@ -48,29 +48,33 @@ Kubernetes cluster, one that meets the following requirements:
 
 .. rst-class:: open
 
-* It must contain at least three nodes (for high availability);
+* It must contain at least three nodes (for high availability).
+  You can also run development workloads on a single-node cluster. Note,
+  however, that you will only be able to provision single-node
+  CrateDB "clusters";
```
Comment (matthijskrul) [inline docs/edge.rst]: You can wrap "CrateDB" inside the previous line

File: docs/edge.rst [hunk 11]
```diff
@@ -48,29 +48,33 @@ Kubernetes cluster, one that meets the following requirements:
...
 
 * Sufficient CPU per node to run the CrateDB Cloud software stack and the OS
   (we recommend at least 4 CPU cores for reliable performance);
 
 * A Kubernetes version > 1.15;
 
-* TCP/443 nginx, TCP/4200, TCP/5432 pointing to Kubernetes;
+* A Kubernetes Load Balancer for accessing CrateDB Clusters;
```
Comment (matthijskrul) [inline docs/edge.rst]:
```suggestion
* A Kubernetes load balancer for accessing CrateDB Clusters;
```

File: docs/edge.rst [hunk 12]
```diff
@@ -306,6 +284,11 @@ Once the cluster is fully deployed, it can be accessed through the `CrateDB
 Admin UI`_ using the username and password you have defined and the URL of your
 cluster.
 
+.. NOTE::
+    If your Kubernetes cluster does not provide a load balancer with an external IP
+    address, you will not be able to access your cluster from the CrateDB Cloud Console.
```
Comment (matthijskrul) [inline docs/edge.rst]: Lines still too long

**Review by matthijskrul — APPROVED** [review]

**Pull request merged**